packages/api/src/lib/suggestedOperations.ts

```diff
@@ -124,3 +124,12 @@ export async function buildCodeSnippetForOperation(oas: Oas, operation: Operatio
 
   return snippet;
 }
+
+export async function getExampleCodeSnippet(oas: Oas, identifier: string) {
```

Comment:
i moved this to here since it was an easy refactor, but i feel like it should live in the `CodeGenerator` class or somewhere where we're not running this code twice during codegen... any ideas @erunion?

Reply:
Yeah this should probably be an instance method somewhere on the `CodeGenerator` class. Maybe it saves the result to a `codesnippet` property that we can re-retrieve if a snippet hasnt been made yet so we don't need to make it twice.